### PR TITLE
Move the dev cache location to /tmp/eb_dev_cache

### PR DESCRIPTION
**Pull request by MKodde**
This to improve the application performance when clearing and warming up the dev caches.

https://www.pivotaltracker.com/story/show/169144846

### CODE REVIEW

File: app/AppKernel.php [hunk 1]
```diff
@@ -62,6 +62,12 @@ public function registerContainerConfiguration(LoaderInterface $loader)
 
     public function getCacheDir()
     {
+        // In the dev environment use a folder outside the shared filesystem. This greatly improves cache clear
+        // and warmup time.
+        if ($this->getEnvironment() === 'dev') {
```
Comment (pablothedude) [inline app/AppKernel.php]: This is maybe something to include in the README.md or maybe even better:
`https://github.com/OpenConext/OpenConext-engineblock/wiki/Development-Guidelines`

**Comment by pablothedude** [comment]
Nice addition!

**Comment by MKodde** [comment]
Thanks for the review

> And what about test?

I was reluctant in moving the cache folder for the test env, this mainly because we can not write to that folder in Travis builds. I'm going to run a couple of experiments nonetheless.

File: app/AppKernel.php (hunk 1 above)
Comment (MKodde) [reply app/AppKernel.php]: Readme did not provide an ample section to add this to, but adding it to the WIKI is a good idea. Adding it!

**Comment by MKodde** [comment]
The test cache seems to work fine on Travis :fireworks:

File: app/AppKernel.php (hunk 1 above)
Comment (pablothedude) [reply app/AppKernel.php]: :100:

**Pull request merged**